int32 index
float32 steer_angle
float32 cur_dist
float32 tar_dist
float32 tar_vel
float32 est_vel
float32 preceding_truck_vel
bool fi_encoder
bool alpha

